website/src/controllers/api-v3/groups.js

```diff
@@ -563,15 +520,25 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
```

Comment:
Should we have a cap on the number of uuids and emails a user can invite at once?